### PR TITLE
Fix window extensions for manifests.

### PR DESCRIPTION
**Pull request by jpmedley**
#### Summary
`onappinstalled` wasn't deprecated. It was moved. Support will continue for `onbeforeinstallprompt`. URLs for both are updated.

#### Test results and supporting details
https://wicg.github.io/manifest-incubations/#extensions-to-the-window-object

### CODE REVIEW

**Comment by jpmedley** [comment]
@Elchi3, This is breaking because it's looking for an MDN URL in the `spec_url` field. Please help.

image

**Comment by Elchi3** [comment]
This is another instance that needs https://github.com/w3c/browser-specs/pull/416

File: api/Window.json [hunk 1]
```diff
@@ -589,13 +590,15 @@
...
             "standard_track": false,
-            "deprecated": true
+            "deprecated": false
           }
         }
       },
       "beforeinstallprompt_event": {
         "__compat": {
           "description": "<code>beforeinstallprompt</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/BeforeInstallPromptEvent",
```
Comment (queengooborg) [inline api/Window.json:600]: This should actually be:

```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/beforeinstallprompt_event",
```

...but this page doesn't exist yet.  A page for `onbeforeinstallprompt` does, however -- so maybe we should link to that for now on the assumption that we'll be merging the two pages in the future?  (cc @Elchi3)

```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/onbeforeinstallprompt",
```

**Review by queengooborg — CHANGES_REQUESTED** [review]
Thanks for opening a PR for this!  I've just got two changes to request:

- We should be linking to the dedicated event page for `beforeinstallprompt`, not its API.  (Suggestions below.)
- Can we also perform the same changes for the event handlers, AKA `onappinstalled` and `onbeforeinstallprompt`?

This won't be mergable yet until the `browser-specs` repo is updated to include the Manifest Incubations specs (and we've bumped the package here in BCD), so I'm going to add a "not ready" label.

**Comment by Elchi3** [comment]
> This won't be mergable yet until the `browser-specs` repo is updated to include the Manifest Incubations specs (and we've bumped the package here in BCD), so I'm going to add a "not ready" label.

I've merged https://github.com/mdn/browser-compat-data/pull/13882. Please merge `main` into this PR and the linter should hopefully pass.

File: api/Window.json (hunk 1 above)
Comment (Elchi3) [reply api/Window.json:600]: https://developer.mozilla.org/docs/Web/API/Window/beforeinstallprompt_event would be the correct link.

https://developer.mozilla.org/docs/Web/API/BeforeInstallPromptEvent is the interface and not the event. 

https://developer.mozilla.org/docs/Web/API/Window/onbeforeinstallprompt is what we're stopping to do.

**Comment by jpmedley** [comment]
@Elchi3 I've updated the spec URLs. Is this in the spec repo yet?

**Comment by Elchi3** [comment]
> @Elchi3 I've updated the spec URLs. Is this in the spec repo yet?

Yes, it is. If you rebase this PR against `main` (or merge `main` into this PR), CI should pass.

**Comment by jpmedley** [comment]
Done.

**Comment by queengooborg** [comment]
Whoa yikes!  What happened with the merge?

I'm going to revert this branch back to the last time relative changes were made and perform a merge to try and fix this.

**Review by queengooborg — APPROVED** [review]
Alright, got the branch fixed now, whew...  This PR is LGTM, thanks Joe!

**Pull request merged**